gui/system/views.py

```diff
@@ -998,7 +998,8 @@ def debug(request):
                 with client as c:
                     c.call('failover.call_remote', 'core.ping')
             except ClientException:
-                return render(request, 'failover/failover_down.html')
+                return render(request, 'system/debug.html', {"failover_availability" : False})
```

Comment:
Shouldn't this rather be as follows:
```
return render(request, 'system/debug.html', {"failover_down" : True})
```

Since in the template, you are checking for: `{% if failover_down %}`